hazelcast/src/main/java/com/hazelcast/sql/impl/extract/QueryTargetDescriptor.java

```diff
@@ -28,7 +30,7 @@
  * The principal difference to {@link QueryTarget} is that descriptor is transferred over a wire, while the target object
  * may contain non-serializable internals.
  */
-public interface QueryTargetDescriptor extends DataSerializable {
+public interface QueryTargetDescriptor extends DataSerializable, Serializable {
```

Comment:
Why `Serializable` is suddenly needed here?

Reply:
Same as below : it is required to serialize `MapScanMetadata` by SQL engine.